test/ffmpeg-vaapi/vpp/brightness.py

```diff
@@ -52,3 +60,29 @@ def compare(k, ref, actual):
     get_media().baseline.check_result(
       compare = compare, context = params.get("refctx", []), psnr = psnr)
 
+@slash.requires(have_ffmpeg)
+@slash.requires(have_ffmpeg_vaapi_accel)
+@slash.requires(*have_ffmpeg_filter("procamp_vaapi"))
+@slash.parametrize(*gen_vpp_brightness_parameters(spec_r2r))
+@platform_tags(VPP_PLATFORMS)
+def test_r2r(case, level):
+  params = init(spec_r2r, case, level)
```

Comment:
Similar to enc/dec r2r, the first `ofile` setup and `call_ffmpeg` can be the same/shared code with both `test_default` and `test_r2r` tests.  If it helps, you can think of the `md5 ref calculation + for loop` as the check metric part of the r2r test.

Reply:
moved to common code one function to avoid duplicate code